flaskbb/forum/forms.py

```diff
@@ -58,12 +58,6 @@ class NewTopicForm(ReplyForm):
     title = StringField(_("Topic title"), validators=[
         DataRequired(message=_("Please choose a title for your topic."))])
 
-    content = TextAreaField(_("Content"), validators=[
-        DataRequired(message=_("You cannot post a reply without content."))])
-
-    track_topic = BooleanField(_("Track this topic"), default=False,
-                               validators=[Optional()])
-
     submit = SubmitField(_("Post Topic"))
 
     def save(self, user, forum):
```

Comment:
Github won't let me comment there, but the call to the `flaskbb_form_new_topic_save` might need to be wrapped in an if depending on if it's actually a new topic or an edit. We could push that into the docs about the hook but that seems like just pushing the burden onto implementers.

Alternative, if the {New,Edit}TopicForm idea above is implemented, the hook can be called before/after a super call.

Reply:
Still have to do this. What do you think about having a hook that is called something like ``flaskbb_form_topic_save`` which is called everytime the topic is saved via the form?